03-specifying-sample-designs.Rmd

```diff
@@ -101,7 +103,7 @@ apisrs_slim <-
 apisrs_slim
 ```
 
-School districts have identifiers `dnum` within counties, and schools have identifiers of `snum` within districts along with their names, `dname` and `sname`. The unique identifier for a school is `cds` which is unique within the state. Additionally, the `fpc` is included as `fpc` and the weight as `pw`. To create the `tbl_survey` object for this SRS data, the design should be specified as follows:
+School districts have identifiers `dnum` within counties, and schools have identifiers of `snum` within districts along with their names, `dname` and `sname`. The unique identifier for a school is `cds` which is unique within the state. Additionally, the `fpc` is included as `fpc` and the weight as `pw`. To create the `tbl_survey` object for this SRS data, the design should be specified as follows: 
```

Comment:
Maybe this would work better as a bulleted list?

Reply:
Made it into a table that I think works well.